ngraph/frontend/paddlepaddle/src/node_context.hpp

```diff
@@ -108,6 +112,17 @@ template <>
 inline ngraph::element::Type NodeContext::get_attribute (const std::string& name, const ngraph::element::Type& def) const
 { return node.get_dtype(name, def); }
 
+inline ngraph::element::Type NodeContext::get_out_port_type(const std::string& port_name) const
+{
+    auto types = get_out_port_types(port_name);
+    PDPD_ASSERT(types.size() > 0, "Port has no tensors connected.");
+    ngraph::element::Type res = types[0];
+    for (const auto& type : types) {
```

Comment:
do we really need this check? If yes, we'll need probably 2 nested 'for' loops to also check types[1]==types[2]?